src/pages/protected/Overview/UniversalObjectCRUD/FormFieldContainers/FormFieldContainerVerordening/FormFieldContainerVerordening.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChangeEventHandler } from 'react'
+
+import { ContainerFormSection } from '@/components/Container'
+import { FormFieldGeldigheid, FormFieldTextInput } from '@/components/Form'
+
+export interface FormFieldContainerVerordeningProps {
+    titleSingular: string
+    crudObject: any
+    handleChange: ChangeEventHandler
+}
+
+function FormFieldContainerVerordening({
+    titleSingular,
+    crudObject,
+    handleChange,
+}: FormFieldContainerVerordeningProps) {
+    return (
+        <>
+            <ContainerFormSection
+                titel="Algemene informatie"
+                beschrijving="De algemene informatie bevat een duidelijke titel.">
+                <FormFieldTextInput
+                    handleChange={handleChange}
+                    fieldValue={crudObject['Titel']}
+                    dataObjectProperty="Titel"
+                    fieldLabel="Titel"
+                    pValue="Beschrijf in een aantal woorden de titel van dit thema."
+                    titleSingular={titleSingular}
+                />
+            </ContainerFormSection>
+
+            <ContainerFormSection
+                titel="Aanvullende informatie"
+                beschrijving="In deze sectie vragen we aanvullende informatie zoals (indien bekend) de datum van inwerkingtreding.">
+                <div className="flex flex-wrap -mx-3">
+                    <FormFieldGeldigheid
```

Comment:
@Aidenbuis Can this be replaced with the FormikDate field from @pzh-ui/components?

Reply:
Will be tackled in the implement-gebiedsprogramma's branch